source/renderer/app/stores/WalletsStore.js

```diff
@@ -918,13 +918,14 @@ export default class WalletsStore extends Store {
 
   isValidAddress = (address: string) => {
     const { app } = this.stores;
-    const { isTestnet } = app.environment;
+    const { isTestnet, isMainnet } = app.environment;
     const addressGroup = isIncentivizedTestnet
       ? AddressGroup.jormungandr
       : AddressGroup.byron;
-    const chainSettings = isTestnet
-      ? ChainSettings.testnet
-      : ChainSettings.mainnet;
+    const chainSettings =
+      isTestnet || (isIncentivizedTestnet && !isMainnet)
```

Comment:
@tomothespian I don't think there can be a case where `isIncentivizedTestnet ` and `isMainnet` are both true.
Perhaps we should change this if clause to be:

```
const chainSettings = isMainnet ? ChainSettings.mainnet : ChainSettings.testnet;
```